### ISSUE TITLE
 Q&A template population

### ISSUE THREAD

**Issue by soulcharmer** [soulcharmer/pubquiz_generator#4]
each question appears on a question slide and then again in on an answer slide.

need to single source the actual text and use some sort of %s to feed the template.

**Comment by soulcharmer** [comment]
Pls add support for a simple and optional markup that cuts off a part of the question not to be repeated in the answer.

**Comment by zelial** [comment]
Currently the question is automatically included in the answer slide without it having to be repeated in the output. @soulcharmer, would you like to change this to _not_ include the question unless the specific token (e.g. %question%) is present in the answer?

**Comment by soulcharmer** [comment]
As shown in #2 , the most flexible and useful solution is to reuse the question text in answer slides as well as the answer sheet by default but also allow the question text to be trimmed for brevity by using a cut-off sequence.

Would it be easy to do?

**Comment by zelial** [comment]
yeah, likely would be easy. Any preferences of how should the cut-off sequence look like in the source?

**Comment by zelial** [comment]
And I'll answer myself: the token should be "<<<" as proposed in issue#2
